classes/02class/exercises/c02-network03/afaryy/README.md

```diff
@@ -0,0 +1,63 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```CLI
```

Comment:
The `CLI` keyword there in markdown tells which format this block of text has.

It is common to use `bash`, `shell script` or `sh` ... or you can leave it empty.

No harm in leaving the `CLI` word there but won't help as well.